processes/annual_data_products.py

```diff
@@ -10,18 +10,20 @@
 
 
 
-class RoutineDataProducts(CubeQueryTask):
+class AnnualDataProducts(CubeQueryTask):
     """
         This process will just extract a product and return it.
     """
 
-    display_name = "Routine Data Products"
+    display_name = "Annual Data Products"
     description = """
-    Access some of the Routine Products from the Data Cube
+    Provides an average cloud-free image of surface reflectance over a time period.
```

Comment:
So this description is only for Geomedian - not for all the Annual data products..... Geomedian is currently the only one so this is not wrong, but when there are more it will need to be changed.

Also for geomedian this is the description on the word doc:
'The geomedian is a multi-dimensional median of surface reflectance for each of the spectral measurements or bands over a time period, it can be used to obtain a cloud-free image' 

Sorry this was probably updated after you did this.